packages/prerender/package.json

```diff
@@ -6,13 +6,12 @@
   "files": [
     "dist",
     "dist/index.js",
```

Comment:
It should be fine to just specify "dist" as a top level, but we should also include "browserUtils"
```suggestion
    "browserUtils"
```